public/scripts/ebayCall.js

```diff
@@ -0,0 +1,65 @@
+// Parse the response and build an HTML table to display search results
+// function _cb_findItemsByKeywords(root) {
+function _cb_findItemsAdvanced  (root) {
+  var items = root.findItemsAdvancedResponse[0].searchResult[0].item || [];
+  var html = [];
+  html.push('<table width="75%" border="0" cellspacing="0" cellpadding="3" id="ebayTable"><tbody>');
+  for (var i = 0; i < items.length; ++i) {
+    var item     = items[i];
+    var title    = item.title;
+    var pic      = item.galleryURL;
+    var viewitem = item.viewItemURL;
+    if (null != title && null != viewitem) {
+      html.push('<tr><td>' + '<img src="' + pic + '" border="0">' + '</td>' +
+      '<td><a href="' + viewitem + '" target="_blank">' + title + '</a></td></tr>');
+    }
+  }
+  html.push('</tbody></table>');
+  $('#ebay').append(html.join(""));
+}
+
+// Create a JavaScript array of the item filters you want to use in your request
+var filterarray = [
```

Comment:
Stretch goal opportunity: You can create inputs / select boxes / or otherwise on your page to give users the ability to modify how they search Ebay.